test-refined/shared/src/main/scala/zio/test/refined/CharInstances.scala

```diff
@@ -18,30 +18,28 @@ trait CharInstances {
   private val whitespaceChars: Seq[Char] =
     (Char.MinValue to Char.MaxValue).filter(_.isWhitespace)
 
+  val digitGen: Gen[Random, Refined[Char, Digit]]   = numericCharGen.map(value => Refined.unsafeApply(value))
```

Comment:
Some Scaladoc would be great here as well!

Reply:
This place has been moved to another place.

Reply:
This has been moved to `object Gen`